### PR TITLE
Emoji Length Calculation Fix

### PR DESCRIPTION
**Pull request by acupoftee**
There was an issue with emoji padding inside table cells. For example, this was the result when the calculations used the `string-width` module:

Screen Shot 2019-03-12 at 9 52 57 AM

It turned out that `stringWidth(s)` in `utils.js:13` returned the wrong length. The unicode length for the  should be `1`, however `stringWidth("🇯🇵")` returns `2`. 

This was corrected using the unicode-aware module [Stringz](https://github.com/sallar/stringz). This PR is to use Stringz to return the correct emoji length and render the correct output as seen below:

Screen Shot 2019-03-12 at 9 52 37 AM

### CODE REVIEW

**Comment by Turbo87** [comment]
@acupoftee it seems that you used `npm` instead of `yarn`, which unintentionally created a `package-lock.json` file. could you remove that file from the first commit and instead update the `yarn.lock` file?

**Comment by acupoftee** [comment]
Will do

File: package.json [hunk 1]
```diff
@@ -13,8 +13,8 @@
     "test": "test"
   },
   "dependencies": {
-    "object-assign": "^4.1.0",
```
Comment (Turbo87) [inline package.json]: why did you change the `object-assign` dependency?

Comment (acupoftee) [reply package.json]: When I tested the changes, npm was unable to locate the existing dependencies so it required a fresh install of the dependencies listed in the package.json and must have updated that particular dependency. Would you like me to revert to `4.1.0`?

Comment (Turbo87) [reply package.json]: yes, ideally this PR should only contain the changes related to `stringz`

Comment (acupoftee) [reply package.json]: Sounds good. `object-assign` was reverted to `^4.1.0` using `yarn`

**Comment by Turbo87** [comment]
@acupoftee I've had to revert this PR in #103 because we discovered that it was responsible for the test suite failing 😢 

it seems that `stringz` does not handle CJK characters the same way and some of our tests are now failing. do you have time to have a look at that?
